tests/test_redshift.py

```diff
@@ -589,6 +589,22 @@ def test_redshiftcopy(mock_session, credentials, dbapi):
                 (),
             )
         )
+        # no delim
```

Comment:
Any chance we can add some integration tests too? I actually don't have access to a redshift cluster atm. Are you able to run those test on your instance?